google-cloud-dialogflow-cx/src/test/java/com/google/cloud/dialogflow/cx/v3beta1/it/ITSystemTest.java

```diff
@@ -0,0 +1,609 @@
+            .build();
+    UpdateEntityTypeRequest updateEntityTypeRequest =
+        UpdateEntityTypeRequest.newBuilder().setEntityType(entityType).build();
+    EntityType response = entityTypesClient.updateEntityType(updateEntityTypeRequest);
+    assertEquals(2, response.getKindValue());
+  }
+
+  @Test
+  public void getFlowTest() {
+    GetFlowRequest request = GetFlowRequest.newBuilder().setName(flowName).build();
+    Flow flow = flowsClient.getFlow(request);
+    assertFlowDetails(flow);
+  }
+
+  @Test
+  public void listFlowsTest() {
+    ListFlowsRequest request = ListFlowsRequest.newBuilder().setParent(agentName).build();
+    FlowsClient.ListFlowsPagedResponse pagedListResponse = flowsClient.listFlows(request);
+    List<Flow> flows = Lists.newArrayList(pagedListResponse.iterateAll());
+    for (Flow flow : flows) {
+      if (flow.getName().equals(flowName)) {
+        assertFlowDetails(flow);
+      }
+    }
+  }
+
+  @Test
+  public void trainFlowTest() throws ExecutionException, InterruptedException {
+    Empty expectedResponse = Empty.newBuilder().build();
+    Empty response = flowsClient.trainFlowAsync(flowName).get();
+    assertEquals(expectedResponse, response);
+  }
+
+  @Test
+  public void getIntentsTest() {
+    GetIntentRequest request = GetIntentRequest.newBuilder().setName(intentsName).build();
+    Intent intent = intentsClient.getIntent(request);
+    assertIntentDetails(intent);
+  }
+
+  @Test
+  public void listIntentsTest() {
+    ListIntentsRequest request = ListIntentsRequest.newBuilder().setParent(agentName).build();
+    IntentsClient.ListIntentsPagedResponse pagedListResponse = intentsClient.listIntents(request);
+    List<Intent> intents = Lists.newArrayList(pagedListResponse.iterateAll());
+    for (Intent intent : intents) {
+      if (intent.getName().equals(intentsName)) {
+        assertIntentDetails(intent);
+      }
+    }
+  }
+
+  @Test
+  public void updateIntentTest() {
+    Intent intent =
+        Intent.newBuilder()
+            .setName(intentsName)
+            .setDisplayName(DISPLAY_NAME)
+            .setPriority(50)
+            .build();
+    FieldMask updateMask = FieldMask.newBuilder().build();
+    Intent intentResponse = intentsClient.updateIntent(intent, updateMask);
+    assertEquals(50, intentResponse.getPriority());
+  }
+
+  @Test
+  public void getPageTest() {
+    GetPageRequest request = GetPageRequest.newBuilder().setName(pageName).build();
+    Page page = pagesClient.getPage(request);
+    assertPageDetails(page);
+  }
+
+  @Test
+  public void listPagesTest() {
+    ListPagesRequest request = ListPagesRequest.newBuilder().setParent(flowName).build();
+    PagesClient.ListPagesPagedResponse pagedListResponse = pagesClient.listPages(request);
+    List<Page> pages = Lists.newArrayList(pagedListResponse.iterateAll());
+    for (Page page : pages) {
+      if (page.getName().equals(pageName)) {
+        assertPageDetails(page);
+      }
+    }
```

Comment:
the same is here, asserPageDetails() might never be called

Reply:
done